libs/platform/ebpf_epoch.c

```diff
@@ -61,39 +63,34 @@
 
 typedef struct _ebpf_epoch_state
 {
-    int64_t epoch;           // The highest epoch seen by this epoch state.
-    bool active : 1;         // Currently within an entry/exit block.
-    bool timer_armed : 1;    // This state has requested the global timer.
-    bool stale : 1;          // This state has entries that haven't been freed.
-    bool timer_disabled : 1; // Prevent re-arming the timer during shutdown.
+    int64_t epoch; // The highest epoch seen by this epoch state.
```

Comment:
why do we need the `struct` any longer? In future if we need to add another field, we can bring back the `struct`.